app/client/components/accountoverview/accountOverviewCard.tsx

```diff
@@ -406,12 +431,19 @@ export const AccountOverviewCard = (props: AccountOverviewCardProps) => {
                 </div>
               )}
             </>
-          ) : (
-            <ul css={keyValuePairCss}>
-              <li css={keyCss}>Payment</li>
-              <li css={valueCss}>FREE</li>
-            </ul>
           )}
+          {!props.productDetail.isPaidTier &&
+            (isGifted ? (
+              <ul css={keyValuePairCss}>
+                <li css={keyCss}>Payment</li>
+                <li css={valueCss}>Gift redemption</li>
+              </ul>
+            ) : (
+              <ul css={keyValuePairCss}>
+                <li css={keyCss}>Payment</li>
+                <li css={valueCss}>FREE</li>
+              </ul>
+            ))}
```

Comment:
This avoids some repetition...
```suggestion
          {!props.productDetail.isPaidTier && (
              <ul css={keyValuePairCss}>
                <li css={keyCss}>Payment</li>
                <li css={valueCss}>{isGifted ? "Gift redemption" : "FREE"}</li>
              </ul>
            )}
```